lib/suspenders/generators/app_generator.rb

```diff
@@ -233,6 +238,14 @@ def remove_routes_comment_lines
       build :remove_routes_comment_lines
     end
 
+    def generate_all
+      run("spring stop")
```

Comment:
I typed a longer thing, but instead I'll reiterate: I have nothing polite to say about this line, or about Spring.